content/docs/administration/display.md

```diff
@@ -4,7 +4,7 @@ description = "Customise how the site looks for you"
 date = 2022-05-17T16:20:00+00:00
 updated = 2022-05-17T16:20:00+00:00
 draft = false
-weight = 71
+weight = 1
```

Comment:
side thought: I think the list of settings in oms would be better alphabetically ordered

Reply:
Ah ok sure - put them in the same order as the accordian in the app, but alphabetised does look nice 😁

Reply:
[febc5c5](https://github.com/msupply-foundation/msupply_docs/pull/47/commits/febc5c5e36e920cb8fb7e9929c57bdff480ae5ed)

Reply:
thanks - I think we should alphabeticalise them in the app too.. will see if we can sneak that in 🤫